packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx

```diff
@@ -154,6 +154,26 @@ const TextCointainer = styled.div`
     height: 30px;
 `;
 
+const SessionCountdownTimer = ({ sessionDeadline }: { sessionDeadline: number }) => {
```

Comment:
I think this is super confusing, since the loading also [indicates](https://github.com/trezor/trezor-suite/pull/7534/commits/bec9e638d415196ed23bf974593d4154229588c5#diff-1e516affede951ba971076e32048520b6cb0000085cf72104fb34170eb23da37L303) initialization of a session right now. I would rather change it to 'soon' as a temporary solution. Absolutely agree that `0 seconds` was never optimal, let's solve it as a part of [#7442](https://github.com/trezor/trezor-suite/issues/7442)?

Reply:
of course ideally it should not happen at all and I'm not sure why it's happening - maybe because of this issue? https://github.com/trezor/trezor-suite/issues/7471

@hynek-jina let's discuss the temporary solution please

Reply:
`0 seconds` to `almost there` should be already done here, right?
https://github.com/trezor/trezor-suite/issues/6909

Reply:
This fixup b8b973891a8f171a783e365674eb4a07a1db94d3 changes to logic of session countdown to never show less than `roundsDurationInHours`. 

this commit [feat(coinjoin): prevent 0 seconds left in status and show loader instead](https://github.com/trezor/trezor-suite/pull/7534/commits/bec9e638d415196ed23bf974593d4154229588c5) should be skipped